### PR TITLE
DS-177 Enforce standard license header on all files and in dist folder

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,13 @@
...
     "build": "vite build",
     "build-tokens": "node design-tokens/build.js",
-    "ts-check": "tsc --noEmit",
-    "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "format": "prettier --write --list-different .",
-    "format-check": "prettier --check ."
+    "format-check": "prettier --check .",
+    "lint": "eslint --ext js,ts,tsx,snap --quiet .",
+    "lint-report": "eslint --ext js,ts,tsx,snap -f json -o build-report/eslint-report.json .",
```
Comment (jeremy-davis-sonarsource) [inline package.json:25]: Should be `build-reports`

File: src/components/radio-button-group/RadioButtonGroup.tsx [hunk 2]
```diff
@@ -18,18 +18,18 @@
...
   // Group Props
   defaultValue?: string;
   disabled?: boolean;
+
   id: string;
+  onChange?: (value: string) => void;
+  options: RadioOption[];
   required?: boolean;
   value?: string;
 }
```
Comment (jeremy-davis-sonarsource) [inline src/components/radio-button-group/RadioButtonGroup.tsx:35]: This breaks the semantic separation that I introduced. Maybe we shouldn't enforce prop ordering?

File: src/generated/design-tokens-light.css [hunk 3]
```diff
@@ -22,11 +22,4 @@
   --card-border-radius: 8px;
   --card-padding: 16px;
   --card-background: #ffffff;
-
-
-  /* Test for RadioButton */
-  --radio-fg: #5D6CD0;
-  --radio-bg: #E8EBFF;
-  --radio-fg-disabled: #C5CDDF;
-  --radio-bg-disabled: #EFF2F9;
```
Comment (jeremy-davis-sonarsource) [inline src/generated/design-tokens-light.css:31]: I know this is just paperclips and duct-tape, but I think we should keep it until we get the actual tokens

**Review by jeremy-davis-sonarsource — CHANGES_REQUESTED** [review]
Looks good, although there are a few things that are breaking the RadioButtonGroup as it currently stands

File: src/components/radio-button-group/RadioButtonGroup.tsx (hunk 2 above)
Comment (gregaubert) [reply src/components/radio-button-group/RadioButtonGroup.tsx:35]: Ah yes, good catch, I'll drop the rule

File: src/generated/design-tokens-light.css (hunk 3 above)
Comment (gregaubert) [reply src/generated/design-tokens-light.css:31]: Yes I'll add it back to be sure the component continues working, we just need to be careful about this when doing a `yarn build-tokens`

**Review by jeremy-davis-sonarsource — APPROVED** [review]

**Pull request merged**